ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -69,11 +69,6 @@ is_chown_forbidden() {
   return $?
 }
 
-is_vboxsf_mountpoint() {
-  grep -q "/app vboxsf" /proc/mounts
-  return $?
-}
-
```

Comment:
I dont think thats a valid solution, although volume mounts might be used in production i'd imagine/hope the root $WORK_DIRECTORY (/app) would only be mounted by developers and if mounting /app in production is needed then the onus should be on that person and not developers to change the environment variable. 

otherwise youre asking developers to add CP specific variables to their bash profiles which if refactored would break those people environments and force them to monitor base image changes...

if you believe people are likely to be mounting /app in production then i can see a few approaches.

1. favour the developer as the person creating production image is more likely to track and follow base image changes so they can set the appropriate variable if they really need to mount /app

2. check the mount for the home|user prefix if present its a developer

3. introduce a PRODUCTION variable, when this is set skip over the smart logic of trying to guess vbox/osxfs/fuse/linux/chown/chmod etc....